### ISSUE TITLE
Do I need to use the proprietary AMD drivers on Ubuntu in order to use ROCM?

### ISSUE THREAD

**Issue by xeon826** [RadeonOpenCompute/ROCm-docker#115]
The directions I see for installing ROCM on Ubuntu involve doing [this](https://github.com/RadeonOpenCompute/ROCm-docker/blob/master/quick-start.md)

```
wget https://repo.radeon.com/amdgpu-install/5.3/ubuntu/focal/amdgpu-install_5.3.50300-1_all.deb 
sudo apt-get install ./amdgpu-install_5.3.50300-1_all.deb
sudo amdgpu-install --usecase=rocm
```

Won’t this install the proprietary AMD graphics drivers? I’d rather not use them because they’re not as good as their open source counterparts on Linux, if I remember correctly when I installed the proprietary AMD drivers for Ubuntu it involved commands similar to this, does the --usecase=rocm make it so that only ROCM is installed?

In a nutshell, my question is if I’m using a Radeon 5700 XT navi11 GPU on Ubuntu 22.04.3 jammy do I need to use the proprietary AMD graphics drivers in order to use ROCM or can I keep the open source drivers? I see there’s [another installation method](https://rocm.docs.amd.com/en/latest/deploy/linux/os-native/install.html) using the Ubuntu repositories would this work to only install ROCM and not the proprietary drivers?

**Comment by kentrussell** [comment]
You can use the upstream kernel with the ROCm userspace stack, but you will be missing some compute-specific features like IPC, RDMA, SPM, etc, as well as the latest kernel+firmware updates. Regardless, if you want to use the stock OS kernel, just pass the --nodkms flag to the amdgpu-install utility.
